roadmap.md

```diff
@@ -22,6 +22,8 @@ v5.0
 
 v4.12 - 2024 Q4
  - Performance improvements
+ - HLS - Pick up an audio track from the original video
```

Comment:
This description doesn't make sense to me.  From this text (also in the PR title), I would not know what changed.

Reply:
This PR allows play the primary audio muxed with the video and also allow the alternate audio in a specific playlist. Do you have any suggestion to a better title?

Reply:
Updated the title and roadmap!